### PR TITLE
[maps] organize helper functions and add tests

### PR DESCRIPTION
**Pull request by adamdoe**
*Description*
- Moves helper functions to helpers folder and adds starter tests
- `cd packages/maps && yarn test` to run tests

### CODE REVIEW

File: packages/map/src/helpers/generateColorsArray.test.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { generateColorsArray } from '../helpers/generateColorsArray'
```
Comment (joshlacey) [inline packages/map/src/helpers/generateColorsArray.test.ts:1]: could you put this in a "tests" subfolder

File: packages/map/src/test/hashObj.test.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { hashObj } from '../helpers/hashObj'
```
Comment (joshlacey) [inline packages/map/src/test/hashObj.test.ts:1]: move to '../helpers/tests' folder

File: packages/map/src/test/generateRuntimeLegendHash.test.ts [hunk 3]
```diff
@@ -0,0 +1,907 @@
+import { generateRuntimeLegendHash } from '../helpers/generateRuntimeLegendHash'
```
Comment (joshlacey) [inline packages/map/src/test/generateRuntimeLegendHash.test.ts:1]: move to '../helpers/tests' folder

File: packages/map/src/test/generateRuntimeLegendHash.test.ts [hunk 4]
```diff
@@ -0,0 +1,907 @@
...
+        {
+          STATE: 'Wyoming',
+          Rate: 43,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        }
+      ],
+      dataFileName: 'valid-data-map.csv',
+      dataFileSourceType: 'file',
+      formattedData: [
```
Comment (joshlacey) [inline packages/map/src/test/generateRuntimeLegendHash.test.ts:537]: can the data be refactored to use fakerjs or to pull the data from some example file

**Review by joshlacey — CHANGES_REQUESTED** [review]

File: packages/map/src/helpers/generateColorsArray.test.ts (hunk 1 above)
Comment (adamdoe) [reply packages/map/src/helpers/generateColorsArray.test.ts:1]: This is the actual function its pulling into the test. Are you suggesting that these functions should be in the test folder as well?

File: packages/map/src/test/generateRuntimeLegendHash.test.ts (hunk 4 above)
Comment (adamdoe) [reply packages/map/src/test/generateRuntimeLegendHash.test.ts:537]: We could use fakerjs, I thought testing a real example after using console logs to get a real hash might be beneficial here to grab the actual hash created. I'm not sure how important that actually is, but we might be able to identify when a real hash has changed/needs to be updated.

File: packages/map/src/helpers/generateColorsArray.test.ts (hunk 1 above)
Comment (joshlacey) [reply packages/map/src/helpers/generateColorsArray.test.ts:1]: yeah create a tests folder in helpers folder and put this generateColorsArray.test.ts file in there. putting the test folder in the folder its testing just reduces clutter for navigating the file tree.

File: packages/map/src/test/generateRuntimeLegendHash.test.ts (hunk 4 above)
Comment (adamdoe) [reply packages/map/src/test/generateRuntimeLegendHash.test.ts]: Used example in examples folder for now.

**Review by joshlacey — APPROVED** [review]

**Pull request merged**